content/03-reference/01-tools-and-interfaces/02-prisma-client/090-raw-database-access.mdx

```diff
@@ -4,20 +4,38 @@ metaTitle: 'Raw database access (Reference)'
 metaDescription: 'Learn how you can send raw SQL queries to your database using the raw() method from the Prisma Client API.'
 ---
 
-## Overview
+<TopBlock>
 
-To send raw SQL queries to your database, you can use two functions that are exposed by your `PrismaClient` instance:
-`queryRaw` and `executeRaw`.
+Prisma Client exposes two methods that allow you to send raw SQL queries to your database:
 
-## `queryRaw`
+- Use `queryRaw` to return actual records (for example, using `SELECT`)
+- Use `executeRaw` to return a count of affected rows (for example, after an `UPDATE` or `DELETE`)
 
-You can use `queryRaw`, if you are interested in retrieving actual database records, for example with `SELECT`:
+Use cases for raw SQL include:
 
-```ts
+- You are a strong SQL user and want to run a heavily optimized query
+- Prisma Client does not yet support the feature that you require (please [consider raising an issue](https://github.com/prisma/prisma/issues/new/choose))
+
+</TopBlock>
+
+## <inlinecode>queryRaw</inlinecode>
+
+`queryRaw` returns actual database records. For example, the following `SELECT` query returns all fields for each record in the `User` table:
+
+```ts no-lines
 const result = await prisma.queryRaw('SELECT * FROM User;')
 ```
 
-`result` is an array where each object corresponds to a retrieved database record:
+The same method is implemented as a tagged template, which makes it easier to use [variables](#using-variables-and-helpers). The following example includes an `${email}` variable:
```

Comment:
tagged template might deserve an external link to some docs that explain what it is